src/main/java/org/datacleaner/extension/elasticsearch/ElasticSearchDocumentIdLookupTransformer.java

```diff
@@ -70,11 +71,18 @@
 
     private UpdateableDatastoreConnection _connection;
 
+    @Validate
+    public void validate() {
+        if (elasticsearchDatastore.getClientType().equals(ElasticSearchDatastore.ClientType.REST)) {
```

Comment:
It would be better to check that it is either NODE or TRANSPORT here. I would formulate it as a switch statement. That way we cover the different scenarios instead of having a blacklist of 1. Should we potentially add more connection modes then this ```if``` would be stale.